### PR TITLE
Instance of named control without passing an element

### PR DESCRIPTION
**Pull request by obaidott**
Added support for creating an instance of a named control without passing an element.

### CODE REVIEW

File: can-control.js [hunk 1]
```diff
@@ -282,11 +282,15 @@ var Control = Construct.extend(
 				pluginname = cls.pluginName || cls.shortName,
 				arr;
 
-			// Retrieve the raw element, then set the plugin name as a class there.
+			if(!element) {
+				dev.warn('can/control/control.js: Creating an instance of a named control without passing an element');
 
+				element = document.createElement('div');
```
Comment (phillipskevin) [inline can-control.js]: do you need to create an element if you're also checking that the element exists below?

File: can-control.js [hunk 2]
```diff
@@ -13,9 +13,9 @@ var assign = require("can-util/js/assign/assign");
 var isFunction = require("can-util/js/is-function/is-function");
 var each = require("can-util/js/each/each");
 var dev = require("can-util/js/dev/dev");
+var vDom = require("can-util/dom/document/document");
```
Comment (justinbmeyer) [inline can-control.js]: this probably shouldn't be called `vDOM`.  There's nothing `v` about it.  All this does is return the document.

**Comment by justinbmeyer** [comment]
What is this supposed to do?  Why would someone do this?  Should it be creating the element on the document (and not an un-attached div)?

**Comment by phillipskevin** [comment]
We ran into [this issue](https://github.com/canjs/can-control/issues/54) while upgrading an app to 3.0. In the app, we just changed the code to not do this, but I don't think it makes sense to throw an error in this case.

**Comment by justinbmeyer** [comment]
@phillipskevin why did it work?  My guess is because jQuery works w/o an element.  If that's the case should we just check that there's no element and exit?  Throwing an error might be reasonable.

**Comment by phillipskevin** [comment]
Right, it worked in 2.3 just because `$(undefined)` returns back a jquery element. I think showing the dev warning and then returning makes sense. I'll re-open the issue.

**Pull request merged**